# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/appstream/hotfixed-honor-docs-flag.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Fix meson.build to honor user supplied -Ddocs flag as mentionned in meson_options.txt.

  - NoTag <notag@t2sde.org>

--- AppStream-1.0.3/meson.build.vanilla	2024-05-02 13:34:42.526357024 +0200
+++ AppStream-1.0.3/meson.build	2024-05-02 13:35:20.579614625 +0200
@@ -223,7 +223,11 @@
 subdir('po/')
 subdir('data/')
 subdir('contrib/')
-subdir('docs/')
+
+if get_option('docs')
+    subdir('docs/')
+endif
+
 subdir('tests/')
 if get_option('qt')
     subdir('qt/')

Disable docs by default and let the user decide.
--- AppStream-1.0.3/meson_options.txt.vanilla	2024-05-02 13:06:40.079733733 +0200
+++ AppStream-1.0.3/meson_options.txt	2024-05-02 13:06:52.394819898 +0200
@@ -57,12 +57,12 @@
 )
 option('apidocs',
        type : 'boolean',
-       value : true,
+       value : false,
        description : 'Build API documentation (automatically enabled when "-Ddocs=true" is passed)'
 )
 option('install-docs',
        type : 'boolean',
-       value : true,
+       value : false,
        description : 'Install documentation for API and specification'
 )
 
